internal/reconciler/status.go

```diff
@@ -5,29 +5,35 @@ const (
 	ReasonReferencedSecretMissing = "ReferencedSecretMissing"
 	ReasonWaitingForLock          = "WaitingForLock"
 
-	ReasonFluentBitDSNotReady = "FluentBitDaemonSetNotReady"
-	ReasonFluentBitDSReady    = "FluentBitDaemonSetReady"
+	ReasonFluentBitDSNotReady          = "FluentBitDaemonSetNotReady"
+	ReasonFluentBitDSReady             = "FluentBitDaemonSetReady"
+	ReasonLogComponentsDeletionBlocked = "LogComponentsDeletionBlocked"
```

Comment:
I think it's not obvious from this naming what is blocking the deletion of what. Maybe we can use something like `LogPipelineBlocksDeletion`. Please think about better ideas.